packages/text/src/TextBase.js

```diff
@@ -18,7 +18,7 @@ const TextBase = styled.span`
   cursor: ${(props) => {
```

Comment:
Instead of always having a cursor property, shall we just not assign it unless the `a` prop or `cursor` prop is defined?

Something like
```javascript
${({ a, cursor }) => {
  if (a || cursor) {
    return `cursor: ${(cursor ? 'cursor' : 'pointer')};`
  }
}}
```

Reply:
Gotcha, sounds good